src/pages/_app.tsx

```diff
@@ -38,7 +38,7 @@ export default function App({ Component, pageProps, layoutProps }) {
   useAnchorLinkAnalytics()
   useEffect(() => makeDevAnalyticsLogger(), [])
 
-  const Layout = Component.layout ?? BaseLayout
+  const Layout = Component.layout ?? EmptyLayout
```

Comment:
The `BaseLayout` is from a stub from ~8 months ago, and is something I think we want to strictly avoid using. With this removal, I've also removed `BaseLayout` from the codebase (we can probably rename `layouts/base-new`, but I'll leave that for another PR!).

I think it's preferable to default to an `EmptyLayout` for now (especially since `_app.tsx` affects dot-io and dev-dot sites).